docs/contributing/documentation/overview.rst

```diff
@@ -71,7 +71,7 @@ To test the documentation before a commit:
 .. code-block:: bash
 
     $ pip install -r docs/requirements.txt
-    # This makes sure that the version of Sphinx you'll get is >=1.4.2!
+    # This makes sure that the version of Sphinx you'll get is >=1.6!
```

Comment:
Can we suggest installation via homebrew also?

Reply:
I think that yes